modules/oe_content_sub_entity/src/Entity/SubEntityBase.php

```diff
@@ -26,9 +28,13 @@ abstract class SubEntityBase extends ContentEntityBase implements SubEntityInter
   /**
    * {@inheritdoc}
    */
-  public function label() {
-    // Return the entity's bundle label by default.
-    return $this->get('type')->entity->label();
+  public function label(bool $update = FALSE) {
+    if (empty($this->label) || $update) {
+      $event = new SubEntityEvent($this);
+      $this->eventDispatcher()->dispatch(SubEntityEvent::LABEL_FORMATION, $event);
+      $this->label = $event->getLabel();
+    }
+    return $this->label;
```

Comment:
We also can add here 
```
  public function save() {
    $this->label = NULL;
    return parent::save();
  }
```
it allows us not to touch modules/oe_content_person/tests/src/Kernel/PersonJobEntityTest.php with `$person_job->label(TRUE);`